templates/pritunl-plugin-ldap.j2

```diff
@@ -0,0 +1,118 @@
+import ldap
+import os
+import pritunlsdk
+from pritunl import logger
+
+# LDAPSearch README
+# https://github.com/chaordic/python-ldap-simple-search
```

Comment:
IMO it could be an docstring for `LDAPSearch` class

Reply:
done